### PR TITLE
TASK: Add release notes for 2.1 and 2.2

### PR DESCRIPTION
**Pull request by kdambekalns**

### CODE REVIEW

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+Search nodes by property & exact value in NodeDataRepository
+============================================================
+
+It is no longer only possible to search through the properties by giving a string that matches for any key or value found in the
+jsonified properties field. Instead, the term can also be an array to match exactly on a given key / value combination. See the `PR #1 <https://github.com/neos/neos-development-collection/pull/1>`_ for details.
+
+Update to latest Font Awesome
+=============================
+
+Neos now ships the Font Awesome, version 4.5.0. This allows integrators to select from a wider range of icons for node types, inspector groups and other uses.
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: it's 4.6.0 now

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+- `Display node label instead of node type label in workspace overview <https://github.com/neos/neos-development-collection/pull/420>`_
+- `Use stable identifier for auto-created child nodes in repair <https://github.com/neos/neos-development-collection/pull/389>`_
+- `Creating pages using non-latin characters <https://github.com/neos/neos-development-collection/pull/394>`_
+- `Node constraints for auto created node types in structure tree <https://github.com/neos/neos-development-collection/pull/405>`_
+- `Image crop selection incorrect on small screens <https://github.com/neos/neos-development-collection/pull/404>`_
+- `Ignore empty NodeType configurations <https://github.com/neos/neos-development-collection/pull/367>`_
+- `Content Collection nodes vanish when publishing to nested workspace <https://github.com/neos/neos-development-collection/pull/372>`_
+- `Event logging tweaks & fixes <https://github.com/neos/neos-development-collection/pull/274>`_
+- `Refresh NodeTree if Workspace is switched <https://github.com/neos/neos-development-collection/pull/334>`_
+- `Reintroduce PublishingService <https://github.com/neos/neos-development-collection/pull/315>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: a regression between 2.1 and 2.2, can be left out..

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+- `Translate process is broken for dimensions without fallbacks <https://github.com/neos/neos-development-collection/pull/392>`_
+- `Editing asset collection <https://github.com/neos/neos-development-collection/pull/476>`_
+- `Introduce interface to customize the default TS prototype <https://github.com/neos/neos-development-collection/pull/473>`_
+- `Add settings to customize login screen stylesheets <https://github.com/neos/neos-development-collection/pull/285>`_
+- `Implement new Login-Screen <https://github.com/neos/neos-development-collection/pull/472>`_
+- `Backend fails to load due to RequireJS timeout <https://github.com/neos/neos-development-collection/pull/463>`_
+- `Use and improve node label <https://github.com/neos/neos-development-collection/pull/448>`_
+- `Only update changed model properties <https://github.com/neos/neos-development-collection/pull/450>`_
+- `Don't show error notification on cancelled page loads <https://github.com/neos/neos-development-collection/pull/449>`_
+- `Aloha table mode issues <https://github.com/neos/neos-development-collection/pull/447>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: a regression between 2.1 and 2.2, can be left out..

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+- `Translate process is broken for dimensions without fallbacks <https://github.com/neos/neos-development-collection/pull/392>`_
+- `Editing asset collection <https://github.com/neos/neos-development-collection/pull/476>`_
+- `Introduce interface to customize the default TS prototype <https://github.com/neos/neos-development-collection/pull/473>`_
+- `Add settings to customize login screen stylesheets <https://github.com/neos/neos-development-collection/pull/285>`_
+- `Implement new Login-Screen <https://github.com/neos/neos-development-collection/pull/472>`_
+- `Backend fails to load due to RequireJS timeout <https://github.com/neos/neos-development-collection/pull/463>`_
+- `Use and improve node label <https://github.com/neos/neos-development-collection/pull/448>`_
+- `Only update changed model properties <https://github.com/neos/neos-development-collection/pull/450>`_
+- `Don't show error notification on cancelled page loads <https://github.com/neos/neos-development-collection/pull/449>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: a regression between 2.1 and 2.2, can be left out..

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+Options for scheme and port have been added to site domains to enable setting them for cross-site linking. Additionally a primary flag to site domains to enable selecting them as the primary domain for a site has been added.
+
+When linking to a node the primary domain of the site the node belongs to is taking into account. This allows for correct cross-site linking instead of creating an invalid link to a non-existing node with the existing site's URL.
+
+Improved thumbnail generation
+=============================
+
+- `Thumbnail generator fixes <https://github.com/neos/neos-development-collection/pull/295>`_
+- `SVG Thumbnail Generator should keep track of width / height <https://github.com/neos/neos-development-collection/pull/289>`_
+- `FontDocumentThumbnailGenerator must respect thumbnail width/height <https://github.com/neos/neos-development-collection/pull/290>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: this is already in 2.1 AFAIK

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+- `Aloha table mode issues <https://github.com/neos/neos-development-collection/pull/447>`_
+- `Improved countByParentAndNodeType performance <https://github.com/neos/neos-development-collection/pull/4>`_
+- `Prevent edit / preview rendering path from applying to front-end <https://github.com/neos/neos-development-collection/pull/429>`_
+- `Correct timezone offset in node info inspector view <https://github.com/neos/neos-development-collection/pull/441>`_
+- `Display node label instead of node type label in workspace overview <https://github.com/neos/neos-development-collection/pull/420>`_
+- `Use stable identifier for auto-created child nodes in repair <https://github.com/neos/neos-development-collection/pull/389>`_
+- `Creating pages using non-latin characters <https://github.com/neos/neos-development-collection/pull/394>`_
+- `Node constraints for auto created node types in structure tree <https://github.com/neos/neos-development-collection/pull/405>`_
+- `Image crop selection incorrect on small screens <https://github.com/neos/neos-development-collection/pull/404>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: this is already in 2.1 AFAIK

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+- `Only update changed model properties <https://github.com/neos/neos-development-collection/pull/450>`_
+- `Don't show error notification on cancelled page loads <https://github.com/neos/neos-development-collection/pull/449>`_
+- `Aloha table mode issues <https://github.com/neos/neos-development-collection/pull/447>`_
+- `Improved countByParentAndNodeType performance <https://github.com/neos/neos-development-collection/pull/4>`_
+- `Prevent edit / preview rendering path from applying to front-end <https://github.com/neos/neos-development-collection/pull/429>`_
+- `Correct timezone offset in node info inspector view <https://github.com/neos/neos-development-collection/pull/441>`_
+- `Display node label instead of node type label in workspace overview <https://github.com/neos/neos-development-collection/pull/420>`_
+- `Use stable identifier for auto-created child nodes in repair <https://github.com/neos/neos-development-collection/pull/389>`_
+- `Creating pages using non-latin characters <https://github.com/neos/neos-development-collection/pull/394>`_
+- `Node constraints for auto created node types in structure tree <https://github.com/neos/neos-development-collection/pull/405>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: this is already in 2.1 AFAIK

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+- `Use and improve node label <https://github.com/neos/neos-development-collection/pull/448>`_
+- `Only update changed model properties <https://github.com/neos/neos-development-collection/pull/450>`_
+- `Don't show error notification on cancelled page loads <https://github.com/neos/neos-development-collection/pull/449>`_
+- `Aloha table mode issues <https://github.com/neos/neos-development-collection/pull/447>`_
+- `Improved countByParentAndNodeType performance <https://github.com/neos/neos-development-collection/pull/4>`_
+- `Prevent edit / preview rendering path from applying to front-end <https://github.com/neos/neos-development-collection/pull/429>`_
+- `Correct timezone offset in node info inspector view <https://github.com/neos/neos-development-collection/pull/441>`_
+- `Display node label instead of node type label in workspace overview <https://github.com/neos/neos-development-collection/pull/420>`_
+- `Use stable identifier for auto-created child nodes in repair <https://github.com/neos/neos-development-collection/pull/389>`_
+- `Creating pages using non-latin characters <https://github.com/neos/neos-development-collection/pull/394>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: this is already in 2.1 AFAIK

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+- `Backend fails to load due to RequireJS timeout <https://github.com/neos/neos-development-collection/pull/463>`_
+- `Use and improve node label <https://github.com/neos/neos-development-collection/pull/448>`_
+- `Only update changed model properties <https://github.com/neos/neos-development-collection/pull/450>`_
+- `Don't show error notification on cancelled page loads <https://github.com/neos/neos-development-collection/pull/449>`_
+- `Aloha table mode issues <https://github.com/neos/neos-development-collection/pull/447>`_
+- `Improved countByParentAndNodeType performance <https://github.com/neos/neos-development-collection/pull/4>`_
+- `Prevent edit / preview rendering path from applying to front-end <https://github.com/neos/neos-development-collection/pull/429>`_
+- `Correct timezone offset in node info inspector view <https://github.com/neos/neos-development-collection/pull/441>`_
+- `Display node label instead of node type label in workspace overview <https://github.com/neos/neos-development-collection/pull/420>`_
+- `Use stable identifier for auto-created child nodes in repair <https://github.com/neos/neos-development-collection/pull/389>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: this is already in 2.0

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+- `Implement new Login-Screen <https://github.com/neos/neos-development-collection/pull/472>`_
+- `Backend fails to load due to RequireJS timeout <https://github.com/neos/neos-development-collection/pull/463>`_
+- `Use and improve node label <https://github.com/neos/neos-development-collection/pull/448>`_
+- `Only update changed model properties <https://github.com/neos/neos-development-collection/pull/450>`_
+- `Don't show error notification on cancelled page loads <https://github.com/neos/neos-development-collection/pull/449>`_
+- `Aloha table mode issues <https://github.com/neos/neos-development-collection/pull/447>`_
+- `Improved countByParentAndNodeType performance <https://github.com/neos/neos-development-collection/pull/4>`_
+- `Prevent edit / preview rendering path from applying to front-end <https://github.com/neos/neos-development-collection/pull/429>`_
+- `Correct timezone offset in node info inspector view <https://github.com/neos/neos-development-collection/pull/441>`_
+- `Display node label instead of node type label in workspace overview <https://github.com/neos/neos-development-collection/pull/420>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: this is already in 2.0

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+- `Use stable identifier for auto-created child nodes in repair <https://github.com/neos/neos-development-collection/pull/389>`_
+- `Creating pages using non-latin characters <https://github.com/neos/neos-development-collection/pull/394>`_
+- `Node constraints for auto created node types in structure tree <https://github.com/neos/neos-development-collection/pull/405>`_
+- `Image crop selection incorrect on small screens <https://github.com/neos/neos-development-collection/pull/404>`_
+- `Ignore empty NodeType configurations <https://github.com/neos/neos-development-collection/pull/367>`_
+- `Content Collection nodes vanish when publishing to nested workspace <https://github.com/neos/neos-development-collection/pull/372>`_
+- `Event logging tweaks & fixes <https://github.com/neos/neos-development-collection/pull/274>`_
+- `Refresh NodeTree if Workspace is switched <https://github.com/neos/neos-development-collection/pull/334>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: this is already in 2.1

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 12]
```diff
@@ -0,0 +1,114 @@
...
+
+All icons are prefixed with "icon-" and old icon-names from version 3.2.1 are still available for backwards compatibility.
+
+Further additions
+=================
+
+In addition to the larger features, lots of small improvements and bugfixes have been included in Neos 2.2, among them:
+
+- `Translate process is broken for dimensions without fallbacks <https://github.com/neos/neos-development-collection/pull/392>`_
+- `Editing asset collection <https://github.com/neos/neos-development-collection/pull/476>`_
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: this is already in 2.0

File: TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst [hunk 13]
```diff
@@ -0,0 +1,114 @@
...
+- `Inspector groups can now be assigned icons <https://github.com/neos/neos-development-collection/pull/455>`_ to help identify them more easily.
+- The structure tree has been improved (`configurable loading depth <https://github.com/neos/neos-development-collection/pull/451>`_, `selectable nodes <https://github.com/neos/neos-development-collection/pull/443>`_, `node type changes <https://github.com/neos/neos-development-collection/pull/442>`_, …).
+- All `dialogs are centered vertically <https://github.com/neos/neos-development-collection/pull/434>`_ now.
+- The Workspaces module has been improved with better date/time handling, styling fixes, …
+- Insert panel groups `are now collapsible <https://github.com/neos/neos-development-collection/pull/418>`_.
+- `Sortable options in selector editors <https://github.com/neos/neos-development-collection/pull/338>`_  (select, references, asset list).
+- Editors can now `switch between sites without logging in again <https://github.com/neos/neos-development-collection/pull/356>`_.
+- Preview uses the `target workspace instead of live <https://github.com/neos/neos-development-collection/pull/399>`_ if applicable.
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst]: would add
- [ ] error notifications have been improved (no more tall orange notifications) (https://github.com/neos/neos-development-collection/pull/424)
- [ ] insert new/paste position selection is more intuitive (opens automatically, only one click needed) – additionally the selected position is shown in the insert new panel (https://github.com/neos/neos-development-collection/pull/426 / https://github.com/neos/neos-development-collection/pull/320)
- [ ] lots of new tooltips and a new way of displaying them (https://github.com/neos/neos-development-collection/pull/320)

**Comment by kdambekalns** [comment]
@aertmann I gave you access to my fork, could you add those things directly? that'd be awesome…

**Comment by aertmann** [comment]
@kdambekalns done

**Comment by kdambekalns** [comment]
Now the redirect handling would deserve to be mentioned…

**Comment by aertmann** [comment]
added notes about the redirect handling

**Pull request merged**